src/browser/animations.js

```diff
@@ -246,10 +246,16 @@ function AnimationHandler(desiredInterval) {
                                 });
                             }
                             hwndToId[hwnd] = bw.id;
+                            if (bw.isMaximized()) {
```

Comment:
you're repeating this 10 times... why not:
```js
bw.demaximize(();
```

And in `Window.create` add:
```js
browserWindow.demaximize = function() {
    if (this.isMaximized()) {
        this.unmaximize();
    }
};
```
If `demaximize` too obscure, how about `unMaximizedIfMaximized`

Reply:
Let's refactor this in a second pass

Reply:
is there a second pass for this?